Research Papers/Shallow learning/README.md

```diff
@@ -261,12 +269,18 @@ This paper investigates a relatively new direction in Multiagent Reinforcement L
 
 <details> <summary> <a href="https://www.econstor.eu/bitstream/10419/95028/1/wp347.pdf"> An Introduction to Evolutionary Game Theory </a>by Weibull, Jörgen W. IFN, 1992. <a href="link">  </a> </summary> Please note that these lecture notes are incomplete and may contain typos and errors. I hope to have a more full-fledged version in a few months, and appreciate in the meantime comments and suggestions.  <br> - </details>
 
 <details> <summary> <a href="http://bobby.cs-i.brandeis.edu/papers/icga5.pdf"> Competitive Environments Evolve Better Solutions for Complex Tasks </a>by Peter J. Angeline, Jordan B. Pollack. ICGA, 1993. <a href="link">  </a> </summary> In the typical genetic algorithm experiment, the fitness function is constructed to be independent of the contents of the population to provide a consistent objective measure. Such objectivity entails significant knowledge about the environment which suggests either the problem has previously been solved or other non-evolutionary techniques may be more efficient. Furthermore, for many complex tasks an independent fitness function is either impractical or impossible to provide. In this paper, we demonstrate that competitive fitness functions, i.e. fitness functions that are dependent on the constituents of the population, can provide a more robust training environment than independent fitness functions. We describe three differing methods for competitive fitness, and discuss their respective advantages <br> - </details>
 
+<details> <summary> <a href="http://www.maths.lse.ac.uk/personal/stengel/TEXTE/geb1996a.pdf"> Efficient Computation of Behavior Strategies </a>by Bernhard von Stengel, 1994. <a href="link">  </a> </summary> We propose the sequence form as a new strategic description for an extensive game with perfect recall. It is similar to the normal form but has linear instead of exponential complexity and allows a direct representation and efficient computation of behavior strategies. Pure strategies and their mixed strategy probabilities are replaced by sequences of consecutive choices and their realization probabilities. A zero-sum game is solved by a corresponding linear program that has linear size in the size of the game tree. General two-person games are studied in the paper by Koller et al., 1996 (Games Econ. Behav. 14, 247–259). Journal of Economic Literature Classification Number: C72. <br> - </details>
```

Comment:
Perhaps this should be under "extensive form games"?

Reply:
Thanks will change it now :raised_hands: